ports/atmel-samd/boards/pewpew_m4/mpconfigboard.mk

```diff
@@ -30,6 +30,7 @@ CIRCUITPY_PWMIO = 0
 CIRCUITPY_RAINBOWIO = 0
 CIRCUITPY_ROTARYIO = 0
 CIRCUITPY_RTC = 0
+CIRCUITPY_SAFEMODE_PY = 0
```

Comment:
Let's leave it on everywhere. As I said in the meeting, this board can be made smaller by turning off terminalio for ru,ja,ko translations where we don't have the font characters anyway. (we do this on samd21 already)

Reply:
@deshipu This would mean adding the below to `pewpew_m4/mpconfigboard.mk` and leaving `CIRCUITPY_SAFEMODE_PY` on. Does this align with what you want for this board?
```
# We don't have room for the fonts for terminalio for certain languages,
# so turn off terminalio, and if it's off and displayio is on,
# force a clean build.
# Note that we cannot test $(CIRCUITPY_DISPLAYIO) directly with an
# ifeq, because it's not set yet.
ifneq (,$(filter $(TRANSLATION),ja ko ru))
CIRCUITPY_TERMINALIO = 0
RELEASE_NEEDS_CLEAN_BUILD = $(CIRCUITPY_DISPLAYIO)
endif
```

Reply:
I have this change in #7497 already too.

Reply:
I pushed the same change from #7497.